src/openforms/js/components/form/edit/tabs.js

```diff
@@ -321,28 +321,108 @@ const PREFILL = {
   ],
 };
 
+const TRANSLATIONS = {
+  key: 'tabs',
+  label: 'Translations',
+  type: 'tabs',
+  components: [
+    {
+      type: 'panel',
+      title: 'NL',
```

Comment:
@sergei-maertens I was looking into making this dynamic (based on the languageInfo endpoint), but I'm not sure if it will work in this fashion. I don't think I can rely on async results (because I would have to fetch the languages from endpoint) when exporting the `TRANSLATIONS` tab.

I also tried turning it into an async function `getTranslationTabs` that generates the tabs, but then I would run into the same issue in other places I think, plus that would mean I have to call the endpoint more often then necessary.

Is there another way to do this?

Reply:
Discussed on Slack, solved by using a `json_script` template filter to pass `settings.LANGUAGES`